src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -174,11 +177,11 @@ public static void SetClusterVersion(int version)
                     break;
                 Interlocked.CompareExchange(ref _currentClusterMinimalVersion, version, currentVersion);
             }
-
-            if (currentVersion != version && Log.IsInfoEnabled)
-            {
+            
+            if (Log.IsInfoEnabled && previousVersion != version) 
                 Log.Info($"Cluster version was changed from {currentVersion} to {version}");
-            }
+
+            AfterClusterVersionSetting?.Invoke(null, new AfterClusterVersionSettingEventArgs(previousVersion, currentVersion));
```

Comment:
why not use `version` ? why in log it will be `version` but in the event you will use different variable? Is there a difference?

Reply:
I've made everything consistent. Now, the `version` value is used everywhere.